rust/oak/build.rs

```diff
@@ -0,0 +1,18 @@
+extern crate protoc_rust;
+
+fn main() {
+    protoc_rust::run(protoc_rust::Args {
+        out_dir: "src/proto",
+        input: &["../../third_party/google/grpc/status.proto"],
```

Comment:
could these be combined as multiple inputs in the same protoc invocation?

Reply:
Yep, done.